plugins/obs-ffmpeg/obs-ffmpeg-srt.h

```diff
@@ -444,7 +446,7 @@ static int libsrt_set_options_pre(URLContext *h, SRTSOCKET fd)
 	}
 	return 0;
 }
-
+#pragma warning(disable : 4996)
```

Comment:
What's the deprecated functionality that's being silenced here?

Reply:
`'inet_addr': Use inet_pton() or InetPton()`

Reply:
i've replaed with inet_pton but i still need to test it, which will be later this weekend

Reply:
tested against a Makito X decoder provided by Haivision. The fix is confirmed working